index.html

```diff
@@ -803,11 +803,9 @@ <h2>Persistence</h2>
   developers.
         </p>
 
-        <p>
-  To avoid these issues, it is RECOMMENDED that <a>DID method</a> specifications
-  only produce <a>DIDs</a> and <a>DID methods</a> bound to strong, stable
-  <a>DID registries</a> capable of making the highest level of commitment to
-  persistence of the <a>DID</a> and <a>DID method</a> over time.
+        <p class="note">
+          To avoid these issues, developers should reference the [[DID-RUBRIC]] to
+          decide which did method best suits the needs of the use case.
```

Comment:
```suggestion
To avoid these issues, developers should refer to the Decentralized Characteristics 
Rubric [[DID-RUBRIC]] to decide which <a>DID method</a> best addresses the needs of 
the use case.
```